system/lib/libc/emscripten_console.c

```diff
@@ -77,3 +78,22 @@ void emscripten_dbg_backtracef(const char* fmt, ...) {
   va_end(ap);
 }
 #endif
+
+void _emscripten_log_formatted(int flags, const char* str);
```

Comment:
You might need to add this to `emscripten_internal.h`

Reply:
Is that for gen_sig_info or?

Reply:
Yup, it for gen_sig_info (it allows us to auto-generate the signature info for all JS function)